docs/upgrade.html

```diff
@@ -59,6 +59,65 @@ <h5><a id="upgrade_350_notable" href="#upgrade_350_notable">Notable changes in 3
         </li>
     </ul>
 
+    <h5><a id="upgrade_350_zk" href="#upgrade_350_zk">Upgrading ZooKeeper-based clusters</a></h5>
+    <p><b>If you are upgrading from a version prior to 2.1.x, please see the note below about the change to the schema used to store consumer offsets.
```

Comment:
"the note below" is rather vague. Can use use an `<a name>` to link to it directly?

Reply:
I changed it to `the note in step 3 below`